src/app/components/common/loading/loading.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="loading-container">
```

Comment:
Can you rebase the loading changes out of your branch please.

Reply:
I honestly don't know why your changes are in here 🤔